qubit-sdk/src/main/java/com/qubit/android/sdk/api/QubitSDK.java

```diff
@@ -67,6 +70,25 @@ public static String getDeviceId() {
     return sdkSingleton.getDeviceId();
   }
 
+  public static void setDeviceId(@Nullable String deviceId) {
```

Comment:
Instead of `setDeviceId` I proposed the name `restartWithCustomDeviceID ` which strongly emphasise what are the effects of calling it.

https://docs.google.com/document/d/1ajKBP1bIg6UCs48ItgiLxMu5iJR_Q-3xqUksuh58III/edit?disco=AAAAcjkzi3I